apple/OmnivoreKit/Sources/Views/SyncingIcon.swift

```diff
@@ -0,0 +1,8 @@
+//
+//  File.swift
+//  
+//
+//  Created by Jackson Harper on 6/5/22.
+//
+
+import Foundation
```

Comment:
code missing or do we need to delete this file?

Reply:
@satindar yeah a little more to come...XCode just added it to the commit accidentally.